docs/scripting/syntax/colors.mdx

```diff
@@ -0,0 +1,108 @@
+| ((%DCHAOS))                                                                                                          | The axiom CHAOS uses the color ((%DD)).                                                                                                                                                                                                        |
+| ((%SCHOICE))                                                                                                         | The axiom CHOICE uses the color ((%SS)).                                                                                                                                                                                                       |
+| ((%QLAW))                                                                                                            | The axiom LAW uses the color ((%qq)).                                                                                                                                                                                                          |
+| ((%YWILD))                                                                                                           | The axiom WILD uses the color ((%YY)).                                                                                                                                                                                                         |
+| {((%Nkey)): ((%Vvalue))}                                                                                             | Object keys use the color ((%NN)), values use the color ((%VV)).                                                                                                                                                                               |
+| ((%D:::TRUST COMMUNICATION:::))                                                                                      | Trust communications use the color ((%DD)).                                                                                                                                                                                                    |
+| ((%C@))((%Juser)), ((%C@))((%Kuser)), ((%C@))((%Muser)), ((%C@))((%Wuser)), ((%C@))((%Luser)), and ((%C@))((%Buser)) | Usernames can use six different colors. The @ uses color ((%CC)), and the name itself can take on any of the colors JKMWLB.                                                                                                                    |
+| ((%A3019))((%BAD)) ((%CD))((%L123))                                                                                  | This date format has four elements. The year is highlighted with color ((%AA)), the day with color ((%LL)). "AD" is colored with color ((%BB)), and "D" is colored with color ((%CC)).                                                         |
```

Comment:
```suggestion
| ((%Sstandard text%))                                                                                                             | The default string color. Uses color ((%SS%)).                                                                                                                                                                                                                                   |
| ((user.script))                                                                                                                  | Script name highlighting. The script author uses color ((%CC%)). The script name uses color ((%LL%)).                                                                                                                                                                            |
| ((trust.script))                                                                                                                 | See above, but for Trust scripts. The user name for Trust scripts are highlighted with color ((%FF%)).                                                                                                                                                                           |
| ((9Q7T199B254M740K991GC))                                                                                                        | GC string highlighting. The numbers use color ((%BB%)), the "((%CGC%))" at the end uses color ((%CC%)). The following colors match to the following denominations: ((%NN%)) -> ((1KGC)), ((%LL%)) -> ((1MGC)), ((%JJ%)) -> ((1BGC)), ((%TT%)) -> ((1TGC)), ((%DD%)) -> ((1QGC)). |
| ((%IVOID_LAMBDA_2%))                                                                                                             | Sectors are automatically highlighted with the sector's axiom color (detailed below).                                                                                                                                                                                            |
| ((%qDATA%))                                                                                                                      | The axiom DATA uses the color ((%qq%)).                                                                                                                                                                                                                                          |
| ((%NKIN%))                                                                                                                       | The axiom KIN uses the color ((%NN%)).                                                                                                                                                                                                                                           |
| ((%lFORM%))                                                                                                                      | The axiom FORM uses the color ((%ll%)).                                                                                                                                                                                                                                          |
| ((%IVOID%))                                                                                                                      | The axiom VOID uses the color ((%II%)).                                                                                                                                                                                                                                          |
| ((%DCHAOS%))                                                                                                                     | The axiom CHAOS uses the color ((%DD%)).                                                                                                                                                                                                                                         |
| ((%FCHOICE%))                                                                                                                    | The axiom CHOICE uses the color ((%FF%)).                                                                                                                                                                                                                                        |
| ((%nLAW%))                                                                                                                       | The axiom LAW uses the color ((%nn%)).                                                                                                                                                                                                                                           |
| ((%CSPC_9999%))                                                                                                                  | "Gray sectors" use the color ((%CC%))                                                                                                                                                                                                                                            |
| ((%EWILD%))                                                                                                                      | The axiom WILD uses the color ((%EE%)).                                                                                                                                                                                                                                          |
| ((key:"value"))                                                                                                                  | Object keys use the color ((%NN%)), values use the color ((%VV%)).                                                                                                                                                                                                               |
| ((%D:::TRUST COMMUNICATION:::%))                                                                                                 | Trust communications use the color ((%DD%)).                                                                                                                                                                                                                                     |
| ((%C@%))((%Juser%)), ((%C@%))((%Kuser%)), ((%C@%))((%Muser%)), ((%C@%))((%Wuser%)), ((%C@%))((%Luser%)), and ((%C@%))((%Buser%)) | Usernames can use six different colors. The ((%C@%)) uses color ((%CC%)), and the name itself can take on any of the colors JKMWLB.                                                                                                                                              |
| ((%A3019%))((%BAD%)) ((%CD%))((%L123%))                                                                                          | This date format has four elements. The year is highlighted with color ((%AA%)), the day with color ((%LL%)). "((%BAD%))" is colored with color ((%BB%)), and "((%CD%))" is colored with color ((%CC%)).                                                                         |

```

Reply:
Big suggest here, but we can leverage our color code plugin's automatic highlighting of client-highlighted strings here. Also, a lot of the axiom colors needed correcting